bundle.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM scratch
```

Comment:
I don't think we need this file, we are using the `bundle-ocp.Dockerfile`

Reply:
agree, will drop it.